schemas/org.openweathermap.snowplow/weather_stamp/jsonschema/1-0-0

```diff
@@ -0,0 +1,151 @@
+{
+  "$schema" : "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "self" : {
+    "vendor" : "org.openweathermap.snowplow",
```

Comment:
I think fixing a dt unix timestamp is within the acceptable bounds of fixing a type rather than anything opinionated, therefore I argue we can change vendor from org.openweathermap.snowplow to org.openweathermap.